docs/languages.md

```diff
@@ -0,0 +1,276 @@
+---
+title: Languages
+---
+
+## Overview
+
+For each language we require a specific structure for your Dockerfile to be able to interpret it and make it run on our platform
+
+
+The Dockerfiles are language specific just like this one:
+
+```dockerfile
+FROM node:8.9.4
+
+RUN mkdir -p app
+
+WORKDIR /app
+
+COPY . /app
+
+RUN npm install
+
+ENV PORT=80
+
+EXPOSE 80
+
+ENTRYPOINT ["npm","run","start"]
+```
+
+However you will find a common behavior:
+
+* Some are using multi-stage build with a base image (with all the dependencies) and a lighter runtime image to run the app
+* You need to export your compiled code or source files in the folder named `app`
+* You will need to include a complementary file (depending on your language) to allow dependencies to be downloaded: like a `Gemfile` for `Ruby` or `requirements.txt` for `Python` with `pip`)
+
+* expose the app on port `80`
+* if the hostname is required by the framework/lib then it has to be set to `0.0.0.0`
+* we would recommend to keep the same `ENTRYPOINT` as the example but depending on the language you can freely modify it
```

Comment:
Saw some do not allow you to modify it? (php below)?.

Reply:
Yup exactly